src/commands/account/faucet.ts

```diff
@@ -0,0 +1,41 @@
+import { Args } from "@oclif/core";
+import { ChainApi, resolveNetworkUrl } from "../../lib/index.js";
+import { AccountData } from "../../types/index.js";
+import { SwankyCommand } from "../../lib/swankyCommand.js";
+import { ConfigError } from "../../lib/errors.js";
+
+export class Faucet extends SwankyCommand<typeof Faucet> {
+  static description = "Faucet some tokens to an account";
+
+  static aliases = [`account:faucet`];
+
+  static args = {
+    alias: Args.string({
+      name: "alias",
+      required: true,
+      description: "Alias of account to be used",
+    }),
+  };
+  async run(): Promise<void> {
+    const { args } = await this.parse(Faucet);
+
+    const accountData = this.swankyConfig.accounts.find(
+      (account: AccountData) => account.alias === args.alias
+    );
+    if (!accountData) {
+      throw new ConfigError("Provided account alias not found in swanky.config.json");
+    }
+
+    const networkUrl = resolveNetworkUrl(this.swankyConfig, "");
+
+    const api = (await this.spinner.runCommand(async () => {
+      const api = await ChainApi.create(networkUrl);
+      await api.start();
+      return api;
+    }, "Connecting to node")) as ChainApi;
+
+
+    await this.spinner.runCommand( async () => {await api.faucet(accountData)}
+    , `Fauceting 100000000000000000000 units to ${args.alias}`);
```

Comment:
Not sure about "Fauceting" as a verb 😅
We can keep it simple: "Transfering ${LOCAL_FAUCET_AMOUNT} units from faucet account to ...."